src/server/engine_shard_set.cc

```diff
@@ -33,6 +33,14 @@ ABSL_FLAG(bool, cache_mode, false,
           "If true, the backend behaves like a cache, "
           "by evicting entries when getting close to maxmemory limit");
 
+ABSL_FLAG(uint32_t, max_eviction_per_heartbeat, 100,
```

Comment:
Instead of defining these flags here and then passing them to DbSlice c'tor - you can just define them in db_slice.cc

Reply:
fixed.